src/test/kotlin/com/pubnub/contract/step/ThenSteps.kt

```diff
@@ -19,14 +19,17 @@ class ThenSteps(private val world: World) {
         assertNotNull(world.pnException)
     }
 
-    @Thens(
-        Then("the result is successful"),
-        Then("I receive a successful response")
-    )
+    @Then("the result is successful")
     fun the_result_is_successful() {
         assertNull(world.pnException)
```

Comment:
Don't we expect success code 200 here?
Wouldn't 
assertEquals(200, world.responseStatus)
be good fit here?

Reply:
That would require changes in the code for other steps not related to those. I'd rather wait with it

Reply:
ok